README.rst

```diff
@@ -683,7 +683,8 @@ Post-rendering and user-induced execution
 
 After rendering has been completed, the browser executes JavaScript code as a result
 of some timing mechanism (such as a Google Doodle animation) or user
-interaction (typing a query into the search box and receiving suggestions).
+interaction (such as typing a query into the search box and receiving suggestions,
+or clicking the microphone or camera icon to switch to searching by audio or images respectively).
 Plugins such as Flash or Java may execute as well, although not at this time on
 the Google homepage. Scripts can cause additional network requests to be
```

Comment:
The request is forwarded to a web server, which stores and delivers web pages in response to client requests.
If the request requires dynamic content, the web server may pass the request to an application server, which processes the request and generates the necessary content.